test/test_lifetime.jl

```diff
@@ -258,6 +258,49 @@ end
     end
 end
 
+@testset "RollingLife - Multiple removal in one sp" begin
+    # Creation and solving of the model
+    ts = TwoLevel(4,10,SimpleTimes(4,1))
+    inv_data = NoStartInvData(
+        FixedProfile(1000),
+        FixedProfile(40),
+        ContinuousInvestment(FixedProfile(0), FixedProfile(15)),
+        RollingLife(StrategicProfile([20,15,25,20]))
+    )
+    demand = StrategicProfile([5,10,15,15])
+    m, para = simple_model(;ts, inv_data, demand)
+
+    # Extraction of required data
+    n = para[:node]
+    𝒯 = para[:T]
+    𝒯ᴵⁿᵛ = strat_periods(𝒯)
+    inv_data = para[:inv_data]
+    disc_rate_1 = 1/(1+para[:disc_rate])^10
+    disc_rate_2 = 1/(1+para[:disc_rate])^20
+    invest = StrategicProfile([5, 5, 15, 0])
+    removal = StrategicProfile([0, 10, 0, 0])
```

Comment:
Perhaps add comment to note the expected double retirement in sp2 due to lifetime extending partly into sp3?